frontend/_apps/shell/pages/api/auth/[...nextauth].ts

```diff
@@ -1,19 +1,10 @@
+import { http } from 'connection';
 import axios from 'axios';
+import { REDIS } from 'connection';
```

Comment:
P1) 그냥 빌드에러가 나요. 에러 메시지는 다음과 같아요. 모듈을 찾을 수 없다고 하는데, 육안으로는 문제가 없어보여요. 다른 부분에서는 잘 작동하는데 `[nextauth]`에서만 오류가 나는거면 next-auth 문제일 수 있을 것 같다는 생각도 들어요.

```
shell:build: Failed to compile.
shell:build: 
shell:build: ./pages/api/auth/[...nextauth].ts
shell:build: Module not found: Can't resolve 'connection'
shell:build: 
shell:build: https://nextjs.org/docs/messages/module-not-found
shell:build: 
shell:build: 
shell:build: > Build failed because of webpack errors
shell:build:  ELIFECYCLE  Command failed with exit code 1.
shell:build: ERROR: command finished with error: command 
```

Reply:
패키지에 진입점 설정이 되지 않아서 생긴 문제였습니다. (package.json의 main key)